api/middleware.go

```
@@ -103,13 +105,22 @@ func errorHandlingMiddleware(w http.ResponseWriter, r *http.Request, next http.H
 	next(w, r)
 	err := context.GetRequestError(r)
 	if err != nil {
+		verbosity, errConv := strconv.Atoi(r.Header.Get(cmd.VerbosityHeader))
```

I think it's safe to simply ignore the error here, verbosity will be 0 anyway.